t/ui/16-activity-view.t

```diff
@@ -94,3 +94,17 @@ subtest 'Current jobs' => sub {
 
 END { kill_driver() }
 done_testing();
+
+# Steps for testing locally
+# This way you will get all combinations of job state and result.
+# Import db dump from o3 or osd
+#   create table job_creations (event_id INT, job_id INT)
+#   insert into job_creations select id as event_id, cast(substr(event_data, 7, 7) as integer) as job_id from audit_events where event = 'job_create' and event_data like '{"id"%';
+# Copy ids from the following select:
+#   select max(jc.event_id) from jobs j INNER join job_creations jc ON j.id=jc.job_id  group by state, result, CASE WHEN blocked_by_id is null THEN '' ELSE 'blocked' END  order by state, result
+# Add this in OpenQA::WebAPI::ServerSideDataTable::render_response():
+#    push @$filter_conds, {
+#        'me.id' => { IN => [
+#                1, 2, 3 # ids from previous select
+#            ]}};
+# Remove encodeURIComponent(currentUser) from openqa.js:renderActivityView()
```

Comment:
It would nevertheless be nice to establish such a setup within an automatic test. I suppose it shouldn't be that difficult to create a bunch of jobs covering all variants of states and results which are possible.

Reply:
The test already creates appropriate events. I would think this is only renevant for manual setup?

Reply:
Since the tests haven't been extended I was assuming this has only been tested manually. My main point is that automatic tests would be nice.